async-context/README.md

```diff
@@ -0,0 +1,360 @@
+|    Host Continuation A()        |
+===================================
+```
+
+and
+
+```
+---------------------------
+| function x()            |
+---------------------------
+| function myLoggingAPI() |
+===================================
+|    continuation f2()            |
+===================================
+|    ...host code...      |
+---------------------------
+|    ...host code...      |
+===================================
+|   Host Continuation B()         |
+===================================
+```
+
+Some notes about the pictures above:
+  - We've introduced a "Root Continuation" as the bottom frame.  This  illustrates a basic assumption that all code is executing in the context of a `Continuation Invocation`.
+  - Multiple `continuations` can be on the stack at the same time, which result in mulitple `Continuation Invocations`. For any given stack frame, the "context" is the first `Continuation Invocation` below the given frame on the stack.
+
+Here, we've updated our pictures of runtime callstacks, giving  labels to the `Continuation Invocation` instances:
+
+```
+---------------------------                    -----------                            
+|   setTimeout()          |                              |
+---------------------------                              |
+|   function f1()         |                              |
+---------------------------                              \/
+|    ...host code...      |                  Continuation Invocation ci1
+---------------------------                              /\  
+|    ...host code...      |                              |
+===================================                      |
+|    Host Continuation A()        |                      |
+===================================            -----------
+```
+
+and
+
+```
+---------------------------                    -----------
+| function x()            |                              |
+---------------------------                              \/                   
+| function myLoggingAPI() |                  Continuation Invocation ci3                        
+===================================                      /\
+|    Continuation f2()            |                      |
+===================================            -----------
+|    ...host code...      |                              |
+---------------------------                              \/
+|    ...host code...      |                   Continuation Invocation ci2                   
+===================================                      /\
+|   Host Continuation B()         |                      |
+===================================            -----------
+```
+
+## Link Context
+We define the `Link Context` `link-c` of a `Continuation` `c` as a pointer from `c` to the `Continuation Invocation` instance where `c` was constructed.  In the example from above, the `continuation` `f2` has a `link context` pointing to `Continuation Invocation` `ci1`.
```

Comment:
It's difficult for me to parse this sentence. I'm not sure how to improve it, but it seems necessary to first mention that there exists some parent/child relationship between continuations, and that the `pointer` refers to an edge illustrating that relationship.

Reply:
tried to re-word.  LMK if it helps.